src/Common/dataAccess/deleteDocument.ts

```diff
@@ -13,7 +13,14 @@ export const deleteDocument = async (collection: CollectionBase, documentId: Doc
     await client()
       .database(collection.databaseId)
       .container(collection.id())
-      .item(documentId.id(), documentId.partitionKeyValue?.length === 0 ? undefined : documentId.partitionKeyValue)
+      .item(
+        documentId.id(),
```

Comment:
NIT: Can we simply this to `documentId.partitionKeyValue?.length === 0 ? "" :  documentId.partitionKeyValue`? Since like the only case we don't want to return `documentId.partitionKeyValue` is when it's an empty array.

Reply:
I thought of doing it this way but then went with the super clear way to illustrate the 3 options. But I like the smaller way better.